tools/release_model.lua

```diff
@@ -39,7 +46,15 @@ local function releaseModule(object, tensorCache)
   if object.release then
     object:release()
   end
-  object:float(tensorCache)
+  object:apply(function (m)
+    for k, v in pairs(m) do
```

Comment:
There is already the same loop a few lines below. The body could be moved there.